lib/logstash/outputs/mongodb.rb

```diff
@@ -67,22 +70,32 @@ def register
             end
           end
         end
+
+        break if @closed.true?
```

Comment:
nitpick: since this is at the tail end of the block passed to `Kernel#loop`, I'd prefer to use either [`until @closed.true? do` ... `end`] or [`while @closed.false? do` ... `end`].

Reply:
ok.